### PR TITLE
Allow comments in .env file when using the env function

### PR DESCRIPTION
**Pull request by Forbidden-A**
Before this pr, using the `env()` function, if you have comments in your file it would give warnings, this pr fixes it.

### CODE REVIEW

File: kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Environment.kt [hunk 1]
```diff
@@ -34,21 +34,29 @@ public fun env(name: String): String? {
             val lines = dotenvFile.readLines()
 
             for (line in lines) {
-                if (!line.contains('=')) {
+                var effectiveLine = line.trimStart()
+                if (effectiveLine.startsWith("#"))
+                    continue
```
Comment (gdude2002) [inline kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Environment.kt:39]: `MultiLineIfElse - [] at .../_Environment.kt:39:21`

tl;dr this needs braces

**Comment by gdude2002** [comment]
It seems you didn't enable the checkbox that allows maintainers to push to the PR branch - we use this for small formatting fixes. Any reason for that?

It's literally a one-line fix, so I'm going to merge anyway and fix it after - but please check that box in future if you can!

**Comment by Forbidden-A** [comment]
> It seems you didn't enable the checkbox that allows maintainers to push to the PR branch - we use this for small formatting fixes. Any reason for that?
> 
> It's literally a one-line fix, so I'm going to merge anyway and fix it after - but please check that box in future if you can!

bruh lmao I always enable it but I must've disabled it by mistake or something cause its enabled by default iirc. Sorry anyways!

**Comment by gdude2002** [comment]
No worries, we got there haha
